libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -871,7 +871,10 @@ export function loadParameterValue(parameter: InputParameter): ValueSegment[] {
   if (parameter.isNotificationUrl) {
     valueObject = `@${constants.HTTP_WEBHOOK_LIST_CALLBACK_URL_NAME}`;
   } else {
-    valueObject = parameter?.value ?? parameter?.default;
+    valueObject = parameter?.value;
+    if (parameter?.required && valueObject === undefined) {
```

Comment:
Curious if you have any examples of this? Just curious since I know we've had feedback from people who expect defaults and I don't recall if the parameters they complained about were optional or not.

Reply:
One such example of this is: https://github.com/Azure/LogicAppsUX/issues/4481
Here we have the optional parameter of `Count` which when the user deletes, we end up repopulating it with a default value of 60.

Reply:
Discussed offline with Eric, was a miscommunication we're still using defaults on creation/adding action, just if the user explicitly deletes the optional parameter do we not use the default parameter